env/build.go

```diff
@@ -37,6 +56,26 @@ func matches(k1, k2 string) bool {
 	return k1 == k2
 }
 
+func isNotIncludedForAPI(platformAPI *api.Version) func(k string) bool {
```

Comment:
```suggestion
func isNotSupported(platformAPI *api.Version) func(k string) bool {
```
Just a suggestion, feel free to ignore (it made more sense to me this way)